Gemfile.lock

```diff
@@ -119,6 +119,9 @@ GEM
       tins (~> 1.6)
     crass (1.0.6)
     date (3.3.4)
+    debug (1.9.2)
```

Comment:
Those I think those deps are introduced by package_json

Reply:
nope, `package_json` has no dependencies outside of core Ruby, and it's not being added as an explicitly dependency anyway (which I think is fine so long as it's only used in the generator - that's why Shakapacker _does_ directly depend on it, but react-rails doesn't)